x-pack/plugins/security_solution/public/common/components/sourcerer/temporary.tsx

```diff
@@ -139,14 +140,15 @@ export const TemporarySourcererComp = React.memo<Props>(
               )}
               {isModified === 'missingPatterns' && (
                 <>
-                  <FormattedMessage
-                    data-test-subj="sourcerer-missing-patterns-callout"
```

Comment:
FormattedMessage does not take `data-test-subj` as a prop, so this is simple ignore when component is rendered. Since the tests were enzyme test, they were passing.